sources/packages/api/src/route-controllers/institution/models/institution.dto.ts

```diff
@@ -126,4 +126,5 @@ export class InstitutionDto extends PartialType(CreateInstitutionDto) {
 export interface InstitutionDetailDto {
   institution: InstitutionDto;
   account: BCeIDDetailsDto;
+  isBCPrivate?: boolean;
```

Comment:
It does not seem as if it could be nullable. If it should not accept null, please remove the "?".

Reply:
As we are not extracting the value in the service layer, we are not setting any value to isBCPrivate. So made it nullable.